src/pages/messages/components/NviApprovals.tsx

```diff
@@ -1,65 +1,38 @@
-import { Box, Skeleton, Typography } from '@mui/material';
+import CheckIcon from '@mui/icons-material/Check';
+import DoNotDisturbIcon from '@mui/icons-material/DoNotDisturb';
+import HourglassEmptyIcon from '@mui/icons-material/HourglassEmpty';
+import { Box, Skeleton, SxProps, Table, TableBody, TableCell, TableHead, TableRow, Typography } from '@mui/material';
 import { useQuery } from '@tanstack/react-query';
 import { useTranslation } from 'react-i18next';
 import { fetchOrganization } from '../../../api/cristinApi';
 import { PublicationPointsTypography } from '../../../components/PublicationPointsTypography';
-import { Approval } from '../../../types/nvi.types';
+import { alternatingNviTableRowColor } from '../../../themes/mainTheme';
+import { Approval, NviCandidateStatus } from '../../../types/nvi.types';
 import { getLanguageString } from '../../../utils/translation-helpers';
 
 interface NviApprovalsProps {
   approvals: Approval[];
-  totalPoints: number;
 }
 
-export const NviApprovals = ({ approvals, totalPoints }: NviApprovalsProps) => {
+export const NviApprovals = ({ approvals }: NviApprovalsProps) => {
   const { t } = useTranslation();
-  const checkedApprovals = approvals.filter(
-    (approval) => approval.status === 'Approved' || approval.status === 'Rejected'
-  );
 
   return (
-    <Box sx={{ m: '1rem', border: '0.5px solid', borderColor: 'nvi.main' }}>
-      <Box
-        sx={{
-          display: 'flex',
-          flexDirection: 'column',
-          justifyContent: 'space-evenly',
-          pb: '0.25rem',
-          bgcolor: 'nvi.main',
-        }}>
-        <Typography fontWeight="bold" sx={{ alignSelf: 'center', my: '0.5rem' }}>
-          {t('tasks.nvi.approval_status_count', { checked: checkedApprovals.length, total: approvals.length })}
-        </Typography>
-
-        {approvals.length > 0 && (
-          <Box
-            sx={{
-              bgcolor: 'white',
-              p: '0.5rem',
-              display: 'grid',
-              gridTemplateColumns: '1fr 3fr 1fr',
-              justifyItems: 'center',
-              gap: '0.5rem',
-            }}>
-            {approvals.map((approvalStatus) => (
-              <InstitutionApprovalStatusRow key={approvalStatus.institutionId} approvalStatus={approvalStatus} />
-            ))}
-          </Box>
-        )}
-      </Box>
-      <Box
-        sx={{
-          display: 'grid',
-          gridTemplateColumns: '1fr 3fr 1fr',
-          bgcolor: 'nvi.light',
-          justifyItems: 'center',
-          px: '0.25rem',
-          my: '0.25rem',
-        }}>
-        <section>=</section>
-        <Typography>{t('tasks.nvi.publication_points')}</Typography>
-        {totalPoints && <PublicationPointsTypography points={totalPoints} />}
-      </Box>
+    <Box sx={{ m: '1rem', border: '1px solid', borderColor: 'nvi.main' }}>
+      <Table size="small" sx={alternatingNviTableRowColor}>
```

Comment:
Om `alternatingNviTableRowColor` bare skal brukes her kunne den godt ligget i denne fila også for min del